src/test/scala/symsim/examples/concrete/cliffwalking/CliffWalkingSpec.scala

```diff
@@ -0,0 +1,38 @@
+package symsim
+package examples.concrete.cliffwalking
+
+import symsim.concrete.Randomized.given
+import CanTestIn.given
+
+import org.scalatest.*
+import prop.*
+import org.scalacheck.Arbitrary.*
+import org.scalacheck.Gen
+import org.scalacheck.Prop.*
+import examples.concrete.cliffwalking.CWState
+import examples.concrete.cliffwalking.CliffWalking
+
+/** Sanity tests for Randomized as a Scheduler */
+class CliffWalkingSpec
+  extends org.scalatest.freespec.AnyFreeSpec,
+    org.scalatestplus.scalacheck.Checkers:
```

Comment:
Rewrite this using pure scalacheck (like in the course). We do not want to introduce scalatest to new files where it is not needed. CC: issue #68